spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    before(:example) { get('/users/745/posts') }
+    it 'check status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders correct template' do
+      expect(response).to render_template('index')
+    end
+
+    it 'check for placeholder text' do
+      expect(response.body).to include 'Posts of user'
+    end
+  end
```

Comment:
- [ ] Test that it does not render a different template
- [ ] Let us change the .to include in this way `expect(response.body).to include("Posts of user")`

Reply:
Hi @Emmyn5600, thanks for mentioning the issues. I have fixed these.

Reply:
These issues have been addressed well. Good Job! :clap: :clap: